packages/wdio-runner/src/index.js

```diff
@@ -139,7 +140,8 @@ export default class Runner extends EventEmitter {
 
         this.reporter.emit('runner:end', {
             failures,
-            cid: this.cid
+            cid: this.cid,
+            retries: failures && retries || 0
```

Comment:
```suggestion
            retries: failures ? retries : 0
```

Reply:
just easier to read

Reply:
Agreed. The ugliness was an artifact from a previous edit. On second thought though, I don't think we should truncate the retries to 0 just because the test passed. It is probably more correct to always pass the correct retry count.
Which do you prefer: (1) `failures ? retries : 0`, (2) `retries || 0`, (3) `retries`.
Personally I would go for (3) as it gives the most information: if you didn't specify retries to begin with, with would be `undefined`. (2) ensures the field is always a number, replacing `undefined` with `0`.